### PR TITLE
[package.xml] Add active maintainers. Much wrong info.

### PR DESCRIPTION
**Pull request by 130s**

### CODE REVIEW

File: package.xml [hunk 1]
```diff
@@ -2,29 +2,31 @@
   <name>srdfdom</name>
   <version>0.3.1</version>
   <description>Parser for Semantic Robot Description Format (SRDF).</description>
-  <author email="isucan@willowgarage.edu">Ioan Sucan</author>
+  <author email="isucan@willowgarage.com">Ioan Sucan</author>
   <author email="gwalck@techfak.uni-bielefeld.de">Guillaume Walck</author>
-  <maintainer email="isucan@willowgarage.com">Ioan Sucan</maintainer>
+  <maintainer email="isucan@gmail.com">Ioan Sucan</maintainer>
```
Comment (davetcoleman) [inline package.xml]: I just emailed @isucan to see if he wants to remain maintainer for packages like this, I'm fine either way

**Comment by davetcoleman** [comment]
This is great, thanks!

File: package.xml (hunk 1 above)
Comment (davetcoleman) [inline package.xml]: he suggested he be removed "from all instances" of package.xmls with him as a maintainer. I think we should also start a separate discussion on Discourse to find new active maintainers for all the packages - want to start that?

**Comment by 130s** [comment]
Comment addressed.

File: package.xml [hunk 2]
```diff
@@ -2,29 +2,30 @@
...
+  <url type="repository">https://github.com/ros-planning/srdfdom</url>
   
   <buildtool_depend>catkin</buildtool_depend>
   <buildtool_depend>pkg-config</buildtool_depend>
 
   <build_depend>boost</build_depend>
+  <build_depend>cmake_modules</build_depend>
   <build_depend>libconsole-bridge-dev</build_depend>
   <build_depend>liburdfdom-headers-dev</build_depend>
+  <build_depend>rostest</build_depend>
```
Comment (v4hn) [inline package.xml]: shouldn't this be a `test_depend` instead?

Comment (davetcoleman) [inline package.xml]: +1

Comment (130s) [inline package.xml]: +1

**Comment by 130s** [comment]
Comment addressed again.
Also noticed that `pkg-config` is not needed (see the commit message for the rationale).

**Pull request merged**